ethportal-api/src/types/consensus/withdrawal.rs

```diff
@@ -2,6 +2,8 @@ use alloy_primitives::Address;
 use alloy_rlp::{RlpDecodable, RlpEncodable};
 use serde::{Deserialize, Deserializer, Serialize};
 
+use crate::consensus::execution_payload::Withdrawal as ConsensusWithdrawal;
```

Comment:
nit: considering that we are in `ethportal-api/src/types/consensus/withdrawal.rs`, wouldn't this alias be a bit confusing? Maybe change it to `ExecutionPayloadWithdrawal`. Or maybe move this file into `ethportal-api/src/types/execution`. What do you think?

I'm not sure why we even have 2 in the first place. I see some difference in deserialization, but I didn't look to check if deserialization is actually the same.

Reply:
> I'm not sure why we even have 2 in the first place. I see some difference in deserialization, but I didn't look to check if deserialization is actually the same.

https://github.com/ethereum/trin/pull/1410#issuecomment-2323925871